ui/src/theme.js

```diff
@@ -298,7 +298,7 @@ const createTheme = isDarkTheme => {
       },
       MuiPickersToolbarText: {
         toolbarTxt: {
-          color: THEME.PRIMARY_TEXT_DARK,
+          color: 'rgba(255, 255, 255, 0.54)',
```

Comment:
This would affect everywhere we use a [material-ui tab](https://material-ui.com/components/tabs/#tabs). We'll want to make this change for the pickers component only.

Reply:
Sorry, possible noob question - I did some digging, but I couldn't find the class `.MuiPickersToolbarText-toolbarTxt` being used by `Tab`/`Tabs` (from [material-ui tab](https://material-ui.com/components/tabs/#tabs)) anywhere.  I looked at the classes used in these tabs, and created this [gist](https://gist.github.com/Manish-Giri/19e0f03351521753b155e9e049edff7a) which has all the classes used by `.MuiTab` and `.MuiTabs`.

Were you referring to the color change in the `PrivateTabIndicator` class here?
```css
  PrivateTabIndicator: {
        colorPrimary: {
          backgroundColor: '#fff',
        },
        colorSecondary: {
          backgroundColor: '#000',
        },
      },
```

Because, I found out that while this changes the color of the tab-indicator underneath the calendar-time dialog, as you had shown in your [comment](https://github.com/taskcluster/taskcluster/pull/2106#issuecomment-563424157), this also changes the color of tab-indicator in other places, like in `http://localhost:5080/auth/scopes`, as shown below -

![Screenshot 2019-12-20 21 38 55](https://user-images.githubusercontent.com/11348778/71301902-0a586d80-2373-11ea-96ea-6f285af948e7.png)

Is this what you meant when you said we should be making the change for the Picker component only?

Reply:
Oh, that's very possible. My mistake.

Reply:
@helfi92 um, so should I try to explore if the color of the TabIndicator can be restricted to the picker component only? For reference, here's how the tabindicator currently looks on https://firefox-ci-tc.services.mozilla.com/auth/scopes -

![Screen Shot 2019-12-23 at 8 22 42 PM](https://user-images.githubusercontent.com/11348778/71387545-6238e380-25c2-11ea-9f17-d6ba5d37f939.png)

But there might be other places too using this tabindicator, which would be affected by this change.

Reply:
Yes, please :)

Reply:
@helfi92 So, I came across an observation while fiddling with the `PrivateTabIndicator` class - the `span` that has this class also has the `MuiTabs-indicator` class -

![Screenshot 2019-12-24 20 30 53](https://user-images.githubusercontent.com/11348778/71428720-91b62180-2690-11ea-989c-cc7c908f5b3a.png)

If I target this class instead of the `PrivateTabIndicator` class, then, while the color of the tab indicator in the picker component remains exactly the way we want it (white in dark theme and black in light theme), this behavior also replicates in other places like `/auth/scopes`, thereby fixing the issue I had pointed out in my comment [here](https://github.com/taskcluster/taskcluster/pull/2106#discussion_r360624987).

In other words, whereas all tab indicators **currently** have the same color (`#4177A5`) irrespective of the theme, as shown in [this comment](https://github.com/taskcluster/taskcluster/pull/2106#discussion_r361039726), with this change, the color should be as follows -

`auth/scopes` in Dark theme -
![Screenshot 2019-12-24 21 12 58](https://user-images.githubusercontent.com/11348778/71428985-413fc380-2692-11ea-918f-d8ba280d3049.png)

`auth/scopes` in Light theme -
![Screenshot 2019-12-24 21 12 49](https://user-images.githubusercontent.com/11348778/71429010-5caace80-2692-11ea-9b01-c2adf4867185.png)

I wanted to check with you if this change is acceptable. I happened to come across this behavior while I was actually looking to target just the `PrivateTabIndicator` class in the picker component, using descendant/child class selectors, but that didn't work out (I'll go into more details on that if you want me to).

Reply:
What do you think of the following change?

```jsx
      MuiPickerDTTabs: {
        tabs: {
          // ...
          '& .MuiTabs-indicator': {
            backgroundColor: isDarkTheme ? '#fff' : '#000',
          },
        },
      },
```

The diff would be:

```diff
diff --git a/ui/src/theme.js b/ui/src/theme.js
index a7f054b8a..2ef178f92 100644
--- a/ui/src/theme.js
+++ b/ui/src/theme.js
@@ -360,14 +360,9 @@ const createTheme = isDarkTheme => {
         tabs: {
           color: '#fff',
           backgroundColor: THEME.SECONDARY,
-        },
-      },
-      PrivateTabIndicator: {
-        colorPrimary: {
-          backgroundColor: '#fff',
-        },
-        colorSecondary: {
-          backgroundColor: '#000',
+          '& .MuiTabs-indicator': {
+            backgroundColor: isDarkTheme ? '#fff' : '#000',
+          },
         },
       },
       MuiListItem: {

```

Reply:
Thank You for the confirmation. I also had something similar, only I didn't think of using `.MuiPickerDTTabs-tabs` as the parent selector!

![Screenshot 2019-12-27 18 33 47](https://user-images.githubusercontent.com/11348778/71536118-f31d0100-28d9-11ea-98e6-9fbb59a2f51c.png)

I'll make the change.